Libraries/LibGfx/Color.h

```diff
@@ -17,7 +17,7 @@
 
 namespace Gfx {
 
-typedef u32 ARGB32;
+typedef u32 BGRA8888;
```

Comment:
On this commit, maybe get a bit more into reasoning, as intuitively `ARGB32` and `BGRA8888` sound like the endian inverses of each other
(also which endiannessis it now?)

Reply:
As with the confusion yelle also mentioned, maybe leave a comment next to the type, like
```c++
// Named after in memory-order (little-endian)
// e.g. 0xRRGGBBAA
```
if I understood the commit and latter comment correctly

Reply:
Added.